janus_core/cli/md.py

```diff
@@ -101,8 +101,8 @@ def md(
         Option(
             help=(
                 """
-                Prefix for output filenames. Default is inferred from structure,
-                ensemble, and temperature.
+                Prefix for output filenames. Default is inferred from structure name.
+                ensemble, temperature, and pressure, if any, are added to this.
```

Comment:
```suggestion
                If any of ensemble, temperature, and pressure are specified they are added to this.
                e.g. NaCL-T200.0.xyz
```
Or something like it